Cargo.toml

```diff
@@ -28,13 +28,13 @@ once_cell = "1.18.0"
 owo-colors = { version = "3.5.0", features = ["supports-colors"] }
 regex = { version = "1.9.3", default-features = false, features = ["perf", "std"] }
 shell-words = "1.1.0"
-strip-ansi-escapes = "0.1.2"
+strip-ansi-escapes = "0.2.0"
```

Comment:
The 0.1.2 release was yanked due to a breaking change and is now uploaded as 0.2.0.